installation/installing-images/mac.md

```diff
@@ -74,7 +74,7 @@ If you specify the wrong device in the instructions, you could overwrite your pr
 
     If the command reports `dd: bs: illegal numeric value`, change the block size `bs=1m` to `bs=1M`.
 
-    If the command reports `dd: /dev/rdiskN: Operation not permitted` you need to disable SIP before continuing.
+    If the command reports `dd: /dev/rdiskN: Operation not permitted` go to  Apple menu -> System Preferences -> Security & Privacy -> Privacy -> Files and Folders -> give Removable Volumes access to Terminal.
```

Comment:
Can you remove whatever the strange character before "Apple menu" is? GitHub is just displaying it as a square.

Reply:
The strnge character is the Apple logo - it displays fine on my Mac! Just delete it - "Apple menu" on its own should be enough.

Reply:
You can access it also from Dock or Spotlight so start with "System Preferences" will be enough.
Deleting ` Apple menu -> `.

Reply:
pushed